catroid/src/main/java/org/catrobat/catroid/content/bricks/SetInstrumentBrick.java

```diff
@@ -0,0 +1,220 @@
+		VOICE_AAHS("Choir Aahs", 53),
+		VOICE_OOHS("Voice Oohs", 54),
+		SYNTH_VOICE("Synth Voice", 55),
+		ORCHESTRA_HIT("Orchestra Hit", 56),
+		TRUMPET("Trumpet", 57),
+		TROMBONE("Trombone", 58),
+		TUBA("Tuba", 59),
+		FRENCH_HORN("French Horn", 61),
+		BRASS_SECTION("Brass Section", 62),
+		SYNTH_BRASS_1("Synth Brass 1", 63),
+		SYNTH_BRASS_2("Synth Brass 2", 64),
+		SOPRANO_SAX("Soprano Sax", 65),
+		ALTO_SAX("Alto Sax", 66),
+		TENOR_SAX("Tenor Sax", 67),
+		BARITONE_SAX("Baritone Sax", 68),
+		OBOE("Oboe", 69),
+		ENGLISH_HORN("English Horn", 70),
+		BASSOON("Bassoon", 71),
+		CLARINET("Clarinet", 72),
+		LEAD_1_SQUARE("Lead Square Wave", 81),
+		LEAD_2_SAWTOOTH("Lead Sawtooth Wave", 82),
+		LEAD_3_CALLIOPE("Lead Calliope", 83),
+		LEAD_4_CHIFF("Lead Chiffer", 84),
+		LEAD_5_CHARANG("Lead Charang", 85),
+		LEAD_6_VOICE("Lead Voice Solo", 86),
+		LEAD_7_FIFTHS("Lead Fifths", 87),
+		LEAD_8_BASS_AND_LEAD("Lead Bass and Lead", 88),
+		PAD_1_NEW_AGE("Pad New Age Fantasia", 89),
+		PAD_2_WARM("Pad Warm", 90),
+		PAD_3_POLYSYNTH("Pad Polysinth", 91),
+		PAD_4_CHOIR("Pad Choir Space Voice", 92),
+		PAD_5_BOWED("Pad Bowed Glass", 93),
+		PAD_6_METALLIC("Pad Metallic Pro", 94),
+		PAD_7_HALO("Pad Halo", 95),
+		PAD_8_SWEEP("Pad Sweep", 96),
+		FX_1_RAIN("FX Rain", 97),
+		FX_2_SOUNDTRACK("FX Soundtrack", 98),
+		FX_3_CRYSTAL("FX Crystal", 99),
+		FX_4_ATMOSPHERE("FX Atmosphere", 100),
+		FX_5_BRIGHTNESS("FX Brightness", 101),
+		FX_6_GOBLINS("FX Goblins", 102),
+		FX_7_ECHOES("FX 7 Echoes, Drops", 103),
+		FX_8_SCI_FI("FX Sci-fi", 104),
+		SITAR("Sitar", 105),
+		BANJO("Banjo", 106),
+		SHAMISEN("Shamisen", 107),
+		KOTO("Koto", 108),
+		KALIMBA("Kalimba", 109),
+		BAGPIPE("Bagpipe", 110),
+		SHANAI("Shanai", 112),
+		TINKLE_BELL("Tinkle bell", 113);
+
+		private String name;
+		private int value;
+		PickableMusicalInstrument(String name, int value) {
+			this.name = name;
+			this.value = value;
+		}
+
+		public static int getIndexByValue(int value) {
+			int index = 0;
+			for (PickableMusicalInstrument instrument : PickableMusicalInstrument.values()) {
+				if (instrument.getValue() == value) {
+					return index;
+				}
+				index++;
+			}
+			return -1;
+		}
+
+		@Override
+		public String getName() {
+			return name;
+		}
+
+		public int getValue() {
+			return value;
+		}
+
+		@Override
+		public void setName(String name) {
+			this.name = name;
+		}
+
+		public void setValue(int value) {
+			this.value = value;
+		}
+	}
+
+	public int instrumentSelection = PickableMusicalInstrument.values()[0].getValue();
+
+	@Override
+	public int getViewResource() {
+		return R.layout.brick_set_instrument;
+	}
+
+	protected int getSpinnerId() {
+		return R.id.set_instrument_spinner;
+	}
+
+	@Override
+	public View getView(Context context) {
+		super.getView(context);
+		List<Nameable> items = new ArrayList<>();
+
+		for (PickableMusicalInstrument instrument : PickableMusicalInstrument.values()) {
+			items.add(instrument);
+		}
```

Comment:
add a new function in `org.catrobat.catroid.utils.java` which returns a list of the constants defined from line 44 to line 133.

Reply:
Because PickableMusicalInstrument is now a class I just create objects of it with the right strings and values, put them all in a list and return it, right?

Reply:
Exactly